unicorecmsmnm/templates/base.pt

```diff
@@ -13,21 +13,28 @@
     <meta name="viewport" content="width=device-width; initial-scale=1.0; maximum-scale=1.0; user-scalable=0;" />
     <meta name="description" content=""/>
     <meta name="keywords" content="malaria, health"/>
-    <link href="${request.static_url('cms:static/css/style.css')}?v=20141014-2" rel="stylesheet" type="text/css" />
+    <link href="${request.static_url('cms:static/css/style.css')}" rel="stylesheet" type="text/css" />
```

Comment:
@SteveBarnett: So whenever I update a CSS file, I should tack the current date on to the end?